Reducing deployment target

tbaranes opened the pull request:
That framework looks really great, but the deployment target is quiet hight for my projects 😬

So, I updated (for carthage and CocoaPods) them to:
- iOS: 8.0
- macOS: 10.0
- tvOS: 9.0 (no need for that one, but I think it's for the best)

Is there any reasons to have higher version? I didn't see any regressions by running the tests / launching the example.

wokalski commented:
Thanks! We actually need to make CI work on those versions, too. In order to accomplish that we need to tweak Travis setup. It is currently being changed in #16 so I'd wait until it's merged.

Thanks again. I'll notify you when the other PR is merged.

wokalski commented:
@tbaranes OK, #16 is merged. Feel free to rebase (or really, it will be easier to just reapply the changes 😜) and update `.travis.yml` (add entries to the [matrix](https://github.com/wokalski/Diff.swift/blob/master/.travis.yml#L21))

tbaranes commented:
Just rebased, I see that the other PR has already reduced the xcodeproj targets, so it will just for CocoaPods!

Travis up to date too, I added all the os destinations supported by `Diff.swift`. Let met know if there's something missing

tbaranes commented:
Any ideas why the iOS10+ targets are failing? 🤔

wokalski commented:
If you view the raw log on Travis you can see that it failed because of a timeout. I saw some discussions about it online but can't find them anymore. @tbaranes.

If you can't find them I'll do my best to fix it over the weekend

tonyarnold commented:
Yeah, you're looking for this: https://github.com/travis-ci/travis-ci/issues/4725

At the bottom of `.travis.yml`, update it to:

```yaml
after_success:
   - bash <(curl -s https://codecov.io/bash)
   - sleep 5 # Workaround for https://github.com/travis-ci/travis-ci/issues/4725
```

tbaranes commented:
Right, I already see that kind of issue on others projects. 
I updated .travis using @tonyarnold answer, thanks guys!

tonyarnold commented on .travis.yml:
```diff
@@ -51,6 +51,6 @@ script:
-
 after_success:
    - bash <(curl -s https://codecov.io/bash)
+   - sleep 5 # Workaround for https://github.com/travis-ci/travis-ci/issues/4725
```
Just a small note - please ensure there are newlines at the end of all of your files.

Otherwise, all 👍

tbaranes replied:
Updated 👌

tbaranes commented:
@wokalski Still one fail due to timeout, can you launch it a second time manually?

wokalski commented:
Awesome @tbaranes. Thanks a lot!
